components/counter/Counter.js

```diff
@@ -6,9 +6,18 @@ import theme from './theme.css';
 
 class Counter extends PureComponent {
   render() {
-    const { children, className, color, count, maxCount, size, ...others } = this.props;
+    const { children, className, color, count, maxCount, size, borderColor, borderTint, ...others } = this.props;
 
-    const classNames = cx(theme['counter'], theme[color], theme[size], className);
+    const classNames = cx(
+      theme['counter'],
+      theme[color],
+      theme[size],
+      {
+        [theme[`border-${borderColor}-${borderTint}`]]: borderTint,
+        [theme[`border-${borderColor}`]]: !borderTint,
```

Comment:
Q: I don't really get this? So if there's a borderTint the classname will be `border-mint-dark` for example, and if there is no borderTint the classname will always be `border-borderColor`? Then you will need a defaultProps for borderColor right? And do we *always* want a borderColor? Or am I just grasping this incorrectly?